package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "redux": "^4.2.0",
     "web-vitals": "^2.1.4"
   },
```

Comment:
- [ ] Please add a license file for your project. You can use these [guidlines](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to create an MIT.md file in your repository which is your license file. :thinking:
- [ ] Don't forget to add your **name**, **current year**, **app type** in your MIT.md file after adding it in your project :+1:
- [ ] Please don't forget to add a link of your MIT.md file in your readme after adding your MIT.md file in your project :blush:

Reply:
done

Reply:
Good Job! :+1: